src/example-types/metricGroups/pivot.js

```diff
@@ -214,14 +194,14 @@ let filter = async ({ filters, groups = [], columns = [] }, schema) => {
           bool: {
             must: [
               ...(await drilldownFilters({
-                drilldowns: filter.groups,
-                groups,
+                drilldowns: filter.rows,
+                rows,
                 schema,
                 getStats,
               })),
               ...(await drilldownFilters({
                 drilldowns: filter.columns,
-                groups: columns,
+                rows: columns,
```

Comment:
the fact that columns are sometimes passed into rows should be clear evidence that this should be called groups internally

Reply:
Fixed